apps/supernova/src/components/regions/RegionsList.tsx

```diff
@@ -17,22 +17,32 @@ const regionsClasses = `
   mb-8
 `
 
-const RegionsList = () => {
+const RegionsList: React.FC = () => {
+  // @ts-ignore
   const { isLoading } = useBoundQuery("alerts")
+  // @ts-ignore
   const severityCountsPerRegion = useAlertsSeverityCountsPerRegion()
+  // @ts-ignore
   const regions = useAlertsRegionsFiltered()
 
-  return (
-    <>
-      {!isLoading && regions && (
+  const renderRegions = () => {
```

Comment:
Is there a reason you extracted the render logic into its own function here? What am I not seeing?

Reply:
Yes, because it was really hard(imposible to write ts-ignore in jsx elements specifically on <>

Reply:
Leave it as it is